agent/acs/handler/acs_handler.go

```diff
@@ -69,6 +70,39 @@ type StartSessionArguments struct {
 	ECSClient            api.ECSClient
 	StateManager         statemanager.StateManager
 	AcceptInvalidCert    bool
+	_time                ttime.Time
```

Comment:
It seems like you can put `_time` and `_timeOnce` and initialization logic into a struct. We can avoid a bunch of duplicated code with that I think.